pkg/api/dashboard.go

```diff
@@ -101,6 +101,12 @@ func GetDashboard(c *middleware.Context) Response {
 		meta.FolderTitle = query.Result.Title
 	}
 
+	if dash.IsFolder {
+		meta.Url = m.GetFolderUrl(dash.Uid, dash.Slug)
```

Comment:
Should do this in 5.1 and keep folder urls as is.

Reply:
Fixed new folder url in frontend_step based on uid, i.e. `/dashboards/f/<uid>/<slug>`.

Reply:
I see this repeated here and in PostDashboard, should it be a function on dashboard instead?

Reply:
Yep, you're right. We should fix this.